### ISSUE TITLE
Multiple Catalogs within one master Catalog [feature request]

### ISSUE THREAD

**Issue by dlewand691** [interactivethings/catalog#421]
I recently discovered Shopify's Polaris documentation and really like how they have different sections for different types of users in a nav bar across the top:

<img width="800" alt="screen shot 2" src="https://user-images.githubusercontent.com/1159913/40071108-e7e3cf8a-583d-11e8-8e51-843b2da6f4f4.png">

[https://polaris.shopify.com/](https://polaris.shopify.com/)

This is a great way to help someone focus on what they're looking for, rather than digging through a long list of links on the left side. Designers just need/want to know about branding, Sketch files, etc. Writers only need/want to know about tone, style, abbreviations, etc, etc etc.

Obviously I could create several different catalogs, one for each type of user, but that'd be a maintenance nightmare. Would it be possible (and would it be of interest) to build this capability into Catalog?

**Comment by herrstucki** [comment]
Yes, improving structure/navigation is something that should and will be done at some point. Thanks for bringing up this use case!

**Comment by dlewand691** [comment]
Hi @herrstucki 
Just following up, checking to see if there might be any progress on this? Thanks!

**Comment by MaluNoPeleke** [comment]
In regard to that it would be nice to have automatically the anchors visible in the left navigation bar.

**Comment by wereHamster** [comment]
This project has been deprecated and is no longer maintained.
